Vagrantfile

```diff
@@ -11,7 +11,6 @@ env_global = [
   'JOBS=4',
   'SHADOW_BUILD_DIR=/tmp/shadow_build_dir',
   'CODESIGN=nocodesign',
-  'secure: RGovyUnMw3fp/bHZi058JvANT1rYmNqrsuSYew0cIgirO6YbMHr/rsjwCm1FTYpBl8s1zgr+u2b8ftYnfnCz2YT+Aip4NWrVYpVU0FEmfytGILrnUS0pjlt8m7fU9AKR1ElOSll7yw7e1kftynN39Q321etvwbLZcXon6zz0suE='
```

Comment:
Should this be part of the commit? If so what implications has it. I never checked the Vagrantfiles

Reply:
This seems to be a syntax error to be - not the : vs = - doesn't build with it there, does everything it needs to without AFAICS.